kedro_kubeflow/generators/utils.py

```diff
@@ -149,3 +150,10 @@ def customize_op(op, image_pull_policy, run_config: RunConfig):
             }
         )
     return op
+
+
+def is_local_fs(filepath):
+    from fsspec.implementations.local import LocalFileSystem
+
+    file_open = fsspec.open(filepath)
```

Comment:
I'm not sure whether opening a file is a good way to check the file system. If the caller does not have permissions to the remote file path, it will fail.

Reply:
According to `fsspec` documentation  FileOpen object doesn't create the underlying file object unless it is used in `with` context.